cli/tests/unit/kv_queue_undelivered_test.ts

```diff
@@ -54,3 +54,14 @@ queueTest("queue with undelivered", async (db) => {
     await listener;
   }
 });
+
+queueTest("throw error if already closed", async (db) => {
+  db.close();
+  await assertThrows(
```

Comment:
Use `assertRejects` here

Reply:
Previously, I used `assertRejects` only, but it got failed 😅.

I don't know, test cases are running fine on my system but failed here.